npc/instances/NydhoggsNest.txt

```diff
@@ -165,6 +165,9 @@ nyd_dun02,100,201,3	script	Yggdrasil Gatekeeper	HIDDEN_NPC,8,8,{
 				close;
 			goto L_Enter;
 		} else {
+			.@instance = has_instance2("1@nyd");
+			if (.@instance >= 0)
```

Comment:
According to our coding style requirements, you use brackets for conditions insider other conditions